sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubsTestEnvironment.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using Azure.Messaging.EventHubs.Core;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   Represents the ambient environment in which the test suite is
+    ///   being run, offering access to information such as environment
+    ///   variables.
+    /// </summary>
+    ///
+    public class EventHubsTestEnvironment: TestEnvironment
+    {
+        public EventHubsTestEnvironment() : base("eventhub")
```

Comment:
"Event Hubs" is the name of the product; we should be using that.

Reply:
This is again the directory name under the sdk folder that Net-TestResource would use a prefix.